src/main/scala/cromwell/binding/WdlBinding.scala

```diff
@@ -219,4 +221,28 @@ case class WdlBinding(ast: Ast) {
   }
 
   def findTask(name: String): Option[Task] = tasks.find(_.name == name)
+
+  /**
+   * Confirm all required inputs are present and attempt to coerce raw inputs to `WdlValue`s.
+   * This can fail if required raw inputs are missing or if the values for a specified raw input
+   * cannot be coerced to the target type of the input as specified in the binding.
+   */
+  def confirmAndCoerceRawInputs(rawInputs: Map[FullyQualifiedName, Any]): Try[Map[FullyQualifiedName, WdlValue]] = {
```

Comment:
I think `coerceRawInputs` would suffice for the name although I suspect you'll push back on this.

To explain my thinking, when I saw the 'and' in the function name I went to look to see if it could be two functions which could then be composed together although it all looks pretty monolithic here. So then I was thinking that if it's hard to tease apart two separate things that there's probably a single description which could be given for the name *shrug*